backend/server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/push/parsing/IssueChangedEventParser.java

```diff
@@ -65,6 +86,8 @@ private boolean isInvalid() {
     private static class ChangedIssuePayload {
       private String issueKey;
       private String branchName;
+      @Nullable
+      private ImpactPayload[] impacts;
```

Comment:
I know it's not on you but I find it very strange that this field is in the `issues` object in the payload and not above

Reply:
Multiple issues may have different impacts, so it cannot be above. Whereas changing the type would modify every issues for example.